README.md

```diff
@@ -33,9 +33,10 @@ Take the following steps to secure your devices and accounts.
 - [ ] Frequently backup your phone and [encrypt your backups](https://support.apple.com/en-us/HT205220)
 
 ## Network security
-- [ ] Find a reputable VPN service with a laptop & mobile phone client to use for hostile networks (e.g. unencrypted wifi) or as an everyday privacy guard
+- [ ] Find a reputable VPN service with a laptop & mobile phone client to use for hostile networks (e.g. unencrypted wifi) or as an everyday privacy guard. [This site](https://thatoneprivacysite.net/vpn-section/) has a great deal of information on different options for VPN providers. Of the providers listed there, [Private Internet Access](https://www.privateinternetaccess.com) is a very good option, and relatively well priced, and provide clients for every major OS, and for android/mac as well.
```

Comment:
I'm solid on adding both the link to That One Privacy Site as well as recommending Private Internet Access. Let's make the language more concise. Can we change

> Find a reputable VPN service with a laptop & mobile phone client to use for hostile networks (e.g. unencrypted wifi) or as an everyday privacy guard. [This site](https://thatoneprivacysite.net/vpn-section/) has a great deal of information on different options for VPN providers. Of the providers listed there, [Private Internet Access](https://www.privateinternetaccess.com) is a very good option, and relatively well priced, and provide clients for every major OS, and for android/mac as well.

to

> Use a reputable VPN service such as [Private Internet Access](https://www.privateinternetaccess.com) to protect yourself in hostile networks (e.g. unencrypted wifi) or as an everyday privacy guard. Don't forget to download and use both their laptop & mobile clients on all of your devices. [That One Privacy Site](https://thatoneprivacysite.net/vpn-section/) also has a comprehensive VPN comparison chart if you'd like to check out other VPN services.

?